scripts/qiita-test-install

```diff
@@ -395,6 +430,8 @@ def main():
         ipython_running = False
         extra_info = ('IPython test failed, perhaps due to an old MOI '
                       'version, please review')
+
+    print qiita_config_header
```

Comment:
it's actually needed so it's displayed, see line https://github.com/biocore/qiita/pull/2118/files#diff-bb7154348182dace785aedbad03c3a2bL341